src/main/features/offer/routes/response.ts

```diff
@@ -0,0 +1,56 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+
+function getPartyStatement (claim: any): any[] {
```

Comment:
Please declare return type correctly. We aim to avoid any. In your case it is `PartyStatement[]`